src/metabase/search/config.clj

```diff
@@ -322,3 +323,12 @@
 (defmethod column->string :default
   [value _ _]
   value)
+
+;;; ----------------------------------------- Helper Fns -----------------------------------------
```

Comment:
What's a helper function? I don't see why this is any more of a "helper function" than `model->alias`, which doesn't appear here.